java/runtime/src/main/java/io/ray/runtime/task/LocalModeTaskSubmitter.java

```diff
@@ -285,8 +285,10 @@ public BaseActorHandle createActor(
     if (options != null) {
       if (options.group != null) {
         PlacementGroupImpl group = (PlacementGroupImpl) options.group;
+        // bundleIndex == -1 indicates using any available bundle.
         Preconditions.checkArgument(
-            options.bundleIndex >= 0 && options.bundleIndex < group.getBundles().size(),
+            options.bundleIndex == -1
+                || options.bundleIndex >= 0 && options.bundleIndex < group.getBundles().size(),
             String.format("Bundle index %s is invalid", options.bundleIndex));
```

Comment:
Could you help to improve the exception message? like `the bundle index xx is illegal, the correct bundle index should be either in the range of 0 to the number of bundles or -1 which means put the task to any available bundles.`

Reply:
Will do.